templates/READMEs/README.md

```diff
@@ -0,0 +1,112 @@
+<!-- TODO: This is an template SDK README.md. It might also be useful for other repository types. Update the TODOs with correct examples/documentation. Replace all instances of "your-repo" with the appropriate name -->
```

Comment:
```suggestion
<!-- TODO: This is a template SDK README.md. It might also be useful for other repository types. Update the TODOs with correct examples/documentation. Replace all instances of "your-repo" with the appropriate name -->
```